apps/balancer-tools/src/app/apr/api/(utils)/validate.ts

```diff
@@ -96,8 +86,18 @@ export const QueryParamsSchema = z
     types: OptionalNullableStringArray.refine(isValidPoolType, {
       message: "Invalid pool type",
     }),
-    minTvl: OptionalNullableFloat,
-    maxTvl: OptionalNullableFloat,
+    minTvl: OptionalNullableFloat.refine(
+      (value) => value === null || (value !== null && value >= 10000),
+      {
+        message: "minTvl should be at least 10,000",
+      },
+    ),
+    maxTvl: OptionalNullableFloat.refine(
+      (value) => value === null || (value !== null && value <= 10000000000),
+      {
+        message: "maxTvl should not be greater than 10,000,000,000",
+      },
+    ),
```

Comment:
why add these validations here? They'll only get in the way IMO 😅

Reply:
https://github.com/bleu-fi/balancer-tools/pull/418#discussion_r1360999805 hehehe sorry